tests/components/template/snapshots/test_weather.ambr

```diff
@@ -0,0 +1,125 @@
+# serializer version: 1
+# name: test_forecast_invalid[config0-1-weather]
+  dict({
+    'forecast': list([
+    ]),
+  })
+# ---
+# name: test_forecast_invalid[config0-1-weather].1
+  dict({
+    'forecast': list([
+    ]),
+  })
+# ---
+# name: test_forecast_invalid_datetime_missing[config0-1-weather]
+  dict({
+    'forecast': list([
+    ]),
+  })
+# ---
+# name: test_forecast_invalid_is_daytime_missing_in_twice_daily[config0-1-weather]
+  dict({
+    'forecast': list([
+    ]),
+  })
```

Comment:
As in, when you're looking at the tests with snapshots, you have to check the `.ambr` file to know what the content is. I think that makes sense when you have to check if a big data structure is the same, not really when you want to check if something is empty

Reply:
(Unless it's a parametrized test imo, in that case I would think an empty snapshot is a valid snapshot

Reply:
The result will look different with the changes I am about to add in #96370:

```python
{"weather.test": { "forecast": []}}
```

vs.

```python
{ "forecast": []}
```

I can add the snapshot in the other PR but it will blow it up even further.

Reply:
Ah right. But then still, I would prefer to only use snapshot tests in places where we need to compare (bigger) data structures. I think this change itself is small enough to be a simple assert instead of making it a snapshot.

My issue is that with snapshot testing, the tests get less readable. And when you're using that with a big structure, it's fine as asserting big structures makes readability even lower.

Reply:
Wait, when I just opened the code I saw your edit, you mean a parametrize?

Reply:
Yes, we can also use add another parameter `expected_response` alongside `service` here: https://github.com/home-assistant/core/pull/96370/files#diff-4f09061ab1ba8b4e1355a0adafdc2d7d1e482648e36fb4b579ef190355403f90R250

I am fine both ways. What do you prefer?

Reply:
If I understand correctly, the result is the same per service? I would then prefer to put it in the parametrize yes. 

(With "Unless it's a parametrized" I mean, if you have a few items that are filled, and one that isn't, that would make sense, but now we're still testing if it's empty)